dotnet/samples/GettingStartedWithProcesses/Utilities/MermaidRenderer.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Reflection;
+using PuppeteerSharp;
+
+namespace Utilities;
+
+/// <summary>
+/// Renders Mermaid diagrams to images using Puppeteer-Sharp.
+/// </summary>
+public static class MermaidRenderer
+{
+    /// <summary>
+    /// Generates a Mermaid diagram image from the provided Mermaid code.
+    /// </summary>
+    /// <param name="mermaidCode"></param>
+    /// <param name="filename"></param>
+    /// <returns></returns>
+    /// <exception cref="InvalidOperationException"></exception>
+    public static async Task GenerateMermaidImageAsync(string mermaidCode, string filename)
```

Comment:
should there be a file extension check somewhere?
I'm not familiar with this lib so don't know what are the supported output extension types - png, jpg, svg, etc

Reply:
puppeteer sharp can generate png and jpg, png by default. Added check for png :)